src/commands/Initialize VS project for VScode.ts

```diff
@@ -0,0 +1,107 @@
+import * as vscode from 'vscode';
+import { window } from 'vscode';
+var path = require('path');
+var find = require('find');
+var fs = require('fs');
+import { generatorProject } from '../yo';
+import { getWorkingFolder } from '../yo';
+import { cleanCsharpApplication } from './clean-application-csharp';
+export async function openVSproject() {
+  var x = await getWorkingFolder();
+  var files = find.fileSync(/\.sfproj$/, x);
+  if (files.length) {
+    var data = fs.readFileSync(files[0]);
+    var isVS = 1;
+  }
+  else {
+    window.showErrorMessage("This project doesn't contain an SFPROJ file.");
+    return;
+  }
+  var appname = path.basename(files[0]).split('.').slice(0, -1).join('.');
+  var services = [];
+  var serviceProjName = new Array();
+  var servicePackage = new Array();
+  var serviceNameStr = new Array();
+  var csprojdata = new Array();
+  var interfaceprojpath = new Array();
+  var csprojinfo;
+  var appPackagePath = appname;
+  var i;
+  var parseString = require('xml2js').parseString;
+  var configjs;
+  await parseString(data, function (err, result) {
+    configjs = result;
+  });
+  var l = 0;
+  for (l = 0; l < configjs["Project"]["ItemGroup"].length; l++) {
+    if (configjs["Project"]["ItemGroup"][l]["ProjectReference"] != undefined)
+      break;
+  }
+  console.log(l);
+  var numofservices = Object.keys(configjs["Project"]["ItemGroup"][l]["ProjectReference"]).length;
+  for (i = 0; i < numofservices; i++) {
+    serviceNameStr[i] = configjs["Project"]["ItemGroup"][l]["ProjectReference"][i]["$"]["Include"];
+  }
+  for (i = 0; i < numofservices; i++) {
```

Comment:
move the code of reading the sfproj and populating the data structure to a new method in this file. Call that method from here.

Reply:
Leave, this for now. Complete other comments first.